src/main/java/ch/wisv/areafiftylan/users/model/User.java

```diff
@@ -100,6 +101,7 @@ public String getPassword() {
 
     public void resetProfile() {
         this.profile = new Profile();
+        this.profile.setBirthday(LocalDate.parse("2000-01-01"));
```

Comment:
This way you're automatically older than 18 after resetting your profile.

Reply:
Yes this was my thought. I would not set this to any date tbh. Maybe the birthday should be nullable then?

Reply:
Maybe localdate.now() makes more sense after all? As @julian9499 implemented initially?

Reply:
Done that @martijnjanssen is this oke?